cpp/node_module/algorithm/node.cpp

```diff
@@ -1 +1,43 @@
 #include "node.hpp"
+
+bool Node::FindRelationship(std::string_view type, mgp::Relationships relationships) {
+  if (type.size() == 0 && relationships.cbegin() != relationships.cend()) {
+    return true;
+  }
+  for (auto relationship : relationships) {
+    if (relationship.Type() == type) {
+      return true;
+    }
+  }
+  return false;
+}
+
+void Node::RelExists(mgp_list *args, mgp_graph *memgraph_graph, mgp_result *result, mgp_memory *memory) {
+  mgp::memory = memory;
+  const auto arguments = mgp::List(args);
+  const auto record_factory = mgp::RecordFactory(result);
+  try {
+    const auto node = arguments[0].ValueNode();
+    const auto pattern = arguments[1].ValueString();
+    bool exists = false;
+
+    if (pattern[0] == '<' && pattern[pattern.size() - 1] == '>') {
+      throw mgp::ValueException("Invalid relationship specification!");
+    }
```

Comment:
This should be consistent with `node.relationship_types` (#300) and `nodes.relationship_types` (#301) - I think the <> case is the same as the bidirectional relationship case.

Reply:
We decided to throw error in this case

Reply:
I see, thanks!